src/components/category-link/category-link.tsx

```diff
@@ -0,0 +1,16 @@
+import { NavLink, NavLinkProps } from '@remix-run/react';
+import { ROUTES } from '~/router/config';
+
+export interface CategoryLinkProps {
+    categorySlug: string;
+    title: string;
```

Comment:
Accepting title rather than children will make it impossible to use this component to wrap images, for example.